backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -292,7 +293,9 @@ public IGitMacheteRepositorySnapshot discoverLayoutAndCreateSnapshot() throws Gi
 
     IGitMacheteRepositorySnapshot createGitMacheteRepository(IBranchLayout branchLayout) throws GitMacheteException {
       var rootBranchTries = branchLayout.getRootEntries().map(entry -> Try.of(() -> createGitMacheteRootBranch(entry)));
-      var rootBranches = Try.sequence(rootBranchTries).getOrElseThrow(GitMacheteException::getOrWrap).toList();
+      var rootBranchesCreationResults = Try.sequence(rootBranchTries).getOrElseThrow(GitMacheteException::getOrWrap).toList();
```

Comment:
Nit: unlike for non-root branches, for root branches each result corresponds to **single** root branch... so maybe better `rootBranchCreationResults`